packages/marko-web-theme-monorail/components/blocks/native-x-promo-card.marko

```diff
@@ -7,6 +8,19 @@ $ const aliases = defaultValue(input.aliases, []);
 $ const blockName = "callout-cards";
 $ const uri = nxConfig.getUri();
 $ const placement = nxConfig.getPlacement({ name: placementName, aliases });
+$ const calculateImageSize = ({ width, height }) => {
+  const aspectRatio = (width / height);
+  if (input.imageWidth && input.imageHeight) {
+    return { width: input.imageWidth, height: input.imageHeight };
+  } else if (input.imageWidth && !input.imageHeight ) {
+    return { width: input.imageWidth, height: input.imageWidth / aspectRatio };
+  } else if (!input.imageWidth && input.imageHeight) {
+    return { width: input.imageHeight * aspectRatio, height: input.imageHeight };
+  }
+  // See https://github.com/parameter1/base-cms/pull/935
+  // For why this is the default
+  return { width: 120, height: (height / (width / 120)) };
```

Comment:
@Shinsina - recommend something like `$ const desiredWidth = defaultValue(input.width, 120);`, so that the default can be overridden if necessary.

Reply:
@brandonbk My thought then would you should be using the already available inputs for `imageWidth` and `imageHeight`